backend/controllers/prisonerProfile/prisonerWorkAndSkills.ts

```diff
@@ -1,26 +1,48 @@
+import e from 'express'
 import logErrorAndContinue from '../../shared/logErrorAndContinue'
+import { capitalize } from '../../utils'
 
 export default ({ prisonerProfileService, esweService }) =>
-  async (req, res) => {
+  async (req: e.Request, res: e.Response): Promise<void> => {
     const { offenderNo } = req.params
 
-    const [prisonerProfileData, functionalSkillLevels, targets, coursesAndQualifications, currentWork] =
-      await Promise.all(
-        [
-          prisonerProfileService.getPrisonerProfileData(res.locals, offenderNo),
-          esweService.getLearnerLatestAssessments(offenderNo),
-          esweService.getLearnerGoals(offenderNo),
-          esweService.getLearnerEducation(offenderNo),
-          esweService.getCurrentActivities(res.locals, offenderNo),
-        ].map((apiCall) => logErrorAndContinue(apiCall))
-      )
+    const [
+      prisonerProfileData,
+      functionalSkillLevels,
+      targets,
+      coursesAndQualifications,
+      currentWork,
+      employabilitySkills,
+    ] = await Promise.all<any>(
+      [
+        prisonerProfileService.getPrisonerProfileData(res.locals, offenderNo),
+        esweService.getLearnerLatestAssessments(offenderNo),
+        esweService.getLearnerGoals(offenderNo),
+        esweService.getLearnerEducation(offenderNo),
+        esweService.getCurrentActivities(res.locals, offenderNo),
+        esweService.getLearnerEmployabilitySkills(offenderNo),
+      ].map((apiCall) => logErrorAndContinue(apiCall))
+    )
+
+    const createEmployabilitySkillsRow = (skill) => ({
```

Comment:
We try to avoid having HTML in JS/TS. This could be achieved by adding to `nunjucksSetup.ts` as a filter, or maybe even as a macro.

One for later.